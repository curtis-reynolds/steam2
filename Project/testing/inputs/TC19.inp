login
admin
logout
login
admin
