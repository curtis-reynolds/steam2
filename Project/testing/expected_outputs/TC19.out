Login successful.
Logout successful.
Login successful.
